skabase/SKALogger/SKALogger/SKALogger.py

```diff
@@ -128,33 +130,86 @@ def Log(self, argin):
         # PROTECTED REGION END #    //  SKALogger.Log
 
     @command(
-    dtype_in='str', 
+    dtype_in='DevVarLongStringArray', 
     doc_in="Central logging level for selected devices", 
     )
     @DebugIt()
     def SetCentralLoggingLevel(self, argin):
         # PROTECTED REGION ID(SKALogger.SetCentralLoggingLevel) ENABLED START #
-        pass
+        CentralLoggingLevel = argin[0][:]
+        CentralLoggingDevice = argin[1][:]
+        i = 0
+        while i < len(CentralLoggingLevel[:]):
+            self.info_stream("%s,%s", CentralLoggingLevel[i], CentralLoggingDevice[i])
+            dev1 = DeviceProxy(CentralLoggingDevice[i])
+            dev1.centralLoggingLevel = CentralLoggingLevel[i]
+            property_names = ["logging_level",
+                              "logging_target",
+                              ]
+            dev_properties = dev1.get_property(property_names)
+            dev_properties["logging_level"] = ["DEBUG"]
+            dev_properties["logging_target"].append("device::central/logger/1")
```

Comment:
Do not use hard-coded strings like "device::central/logger/1". Create a constant at the beginning and use it throughout the code. Ideally, this value should be kept in a config file and read at the time of device initialization.
Similarly, use constants for setting logging levels.

Reply:
Not sure I understand why overwriting these properties in the code. Moreover, suggest to put default values into Class properties (and/ or device Properties) instead of files.

Reply:
Removed the properties from code. Added in the SKABaseDevice as Device Properties (in Pogo).

Reply:
That's in the upcoming commit. Just doing that.